include/circt/Dialect/FIRRTL/FIRRTLDeclarations.td

```diff
@@ -124,12 +124,12 @@ def RegOp : FIRRTLOp<"reg", [/*MemAlloc*/]> {
   }];
 }
 
-def RegInitOp : FIRRTLOp<"reginit", [/*MemAlloc*/]> {
+def RegResetOp : FIRRTLOp<"regreset", [/*MemAlloc*/]> {
   let summary = "Define a new register with an initializer";
   let description = [{
     Declare a new register:
     ```
-      %name = firrtl.reginit %clockVal, %resetSignal, %resetValue : t1
+      %name = firrtl.regreset %clockVal, %resetSignal, %resetValue : t1
     ```
     }];
```

Comment:
The name change feels somewhat unnecessary relative to the functional content of the PR. I'm fine to keep this if you think it's more clear this way.